src/Fields/interpolate.jl

```diff
@@ -68,30 +68,30 @@ end
     x₀ = xnode(1, 1, 1, grid, locs...)
     Δx = xspacings(grid, locs...)
     FT = eltype(grid)
-    return convert(FT, (x - x₀) / Δx)
+    return convert(FT, (x - x₀) / Δx) # 0 - based indexing!
```

Comment:
```suggestion
    return convert(FT, (x - x₀) / Δx) + 1
```

why isn't this the fix?